.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!-- Provide a general summary of your changes in the title above -->
+
+<!--
+  By opening this PR you confirm that will follow the contribution guidelines. You agree to submit a fully featured working manifest that creates shortcuts, bin shims, persists data, enable portable mode, and auto updates.
+-->
+
+[ ] I have read the [Contributing Guide](../CONTRIBUTING.md).
+
+## What emulator release type is this?
```

Comment:
scoop-games contains more than just emulators :slightly_smiling_face: 

```suggestion
## What package release type is this?
```

It also contains launchers and various modding tools, so "game" is too specific as well.